src/i18n/keymap_extras/keymap_spanish_latin_america.js

```diff
@@ -0,0 +1,182 @@
+  KC_TILD: { name: '°', title: 'ES_MORD' },
+  'S(KC_1)': { name: '!', title: 'ES_EXLM' },
+  KC_EXLM: { name: '!', title: 'ES_EXLM' },
+  'S(KC_2)': { name: '"', title: 'ES_DQUO' },
+  KC_AT: { name: '"', title: 'ES_DQUO' },
+  'S(KC_3)': { name: '#', title: 'ES_NUMB' },
+  KC_HASH: { name: '#', title: 'ES_NUMB' },
+  'S(KC_4)': { name: '$', title: 'ES_DLR' },
+  KC_DLR: { name: '$', title: 'ES_DLR' },
+  'S(KC_5)': { name: '%', title: 'ES_PERC' },
+  KC_PERC: { name: '%', title: 'ES_PERC' },
+  'S(KC_6)': { name: '&', title: 'ES_AMPR' },
+  KC_CIRC: { name: '&', title: 'ES_AMPR' },
+  'S(KC_7)': { name: '/', title: 'ES_SLSH' },
+  KC_AMPR: { name: '/', title: 'ES_SLSH' },
+  'S(KC_8)': { name: '(', title: 'ES_LPRN' },
+  KC_ASTR: { name: '(', title: 'ES_LPRN' },
+  'S(KC_9)': { name: ')', title: 'ES_RPRN' },
+  KC_LPRN: { name: ')', title: 'ES_RPRN' },
+  'S(KC_0)': { name: '=', title: 'ES_EQL' },
+  KC_RPRN: { name: '=', title: 'ES_EQL' },
+  'S(KC_MINS)': { name: '?', title: 'ES_QUES' },
+  KC_UNDS: { name: '?', title: 'ES_QUES' },
+  'S(KC_EQL)': { name: '¡', title: 'ES_IEXL' },
+  KC_PLUS: { name: '¡', title: 'ES_IEXL' },
+  // Row 2
+  'S(KC_LBRC)': { name: '¨', title: 'ES_DIAE' },
+  KC_LCBR: { name: '¨', title: 'ES_DIAE' },
+  'S(KC_RBRC)': { name: '*', title: 'ES_ASTR' },
+  KC_RCBR: { name: '*', title: 'ES_ASTR' },
+  // Row 3
+  'S(KC_QUOT)': { name: '[', title: 'ES_LBRC' },
+  KC_DQUO: { name: '[', title: 'ES_LBRC' },
+  // Row 4
+  'S(KC_NUBS)': { name: '>', title: 'ES_RABK' },
+  'S(KC_COMM)': { name: ';', title: 'ES_SCLN' },
+  KC_LT: { name: ';', title: 'ES_SCLN' },
+  'S(KC_DOT)': { name: ':', title: 'ES_COLN' },
+  KC_GT: { name: ':', title: 'ES_COLN' },
+  'S(KC_SLSH)': { name: '_', title: 'ES_UNDS' },
+  KC_QUES: { name: '_', title: 'ES_UNDS' },
+
+  /* AltGr symbols
+   * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+   * │ ¬ │   │   │   │   │   │   │   │   │   │   │ \ │   │       │
+   * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+   * │     │ @ │   │ € │   │   │   │   │   │   │   │   │ ~ │     │
+   * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+   * │      │   │   │   │   │   │   │   │   │   │   │ ^ │ ` │    │
+   * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+   * │    │   │   │   │   │   │   │   │   │   │   │   │          │
+   * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+   * │    │    │    │                        │    │    │    │    │
+   * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+   */
+  // Row 1
+  'ALGR(KC_GRV)': { name: '¬', title: 'ES_NOT' },
+  'ALGR(KC_EQL)': { name: '\\', title: 'ES_BSLS' },
+  // Row 2
+  'ALGR(KC_Q)': { name: '@', title: 'ES_AT' },
+  'ALGR(KC_E)': { name: '€', title: 'ES_EUR' },
+  'ALGR(KC_RBRC)': { name: '~', title: 'ES_TILD' },
+  // Row 3
+  'ALGR(KC_QUOT)': { name: '^', title: 'ES_CIRC' },
+  'ALGR(KC_NUHS)': { name: '`', title: 'ES_ACUT' },
```

Comment:
```suggestion
  'ALGR(KC_NUHS)': { name: '`', title: 'ES_GRV (dead)' },
```

WARNING: `ES_GRV` would be the canonical alias for a dead grave key but it was forgotten in https://github.com/qmk/qmk_firmware/pull/22542 — [see how there is nothing after the declaration of `ES_CIRC`](https://github.com/JuanoD/qmk_firmware/blob/9aa4049d5603c67e7d4db564d25d1b4eab35c235/data/constants/keycodes/extras/keycodes_spanish_latin_america_0.0.1.hjson#L335) — so `ES_GRV` technically does not exist yet in qmk_firmware, and should be added.